src/interface/blas2/backend/intel_gpu.hpp

```diff
@@ -96,5 +96,19 @@ typename sb_handle_t::event_t _tbmv(sb_handle_t& sb_handle, index_t _N,
 }
 }  // namespace backend
 }  // namespace tbmv
+
+namespace trsv {
+namespace backend {
+template <uplo_type uplo, transpose_type trn, diag_type diag,
+          typename sb_handle_t, typename index_t, typename container_t0,
+          typename container_t1, typename increment_t>
+typename sb_handle_t::event_t _trsv(sb_handle_t& sb_handle, index_t _N,
+                                    container_t0 _mA, index_t _lda,
+                                    container_t1 _vx, increment_t _incx) {
+  return blas::internal::_trsv_impl<256, uplo, trn, diag>(sb_handle, _N, _mA,
```

Comment:
Is there a specific reason why you chose a larger work group size for Intel GPU as compared to the AMD GPU backend or others?

Reply:
No, I introduced the backends for future architecture tuning.